app/src/main/java/com/example/duccm/learningapp/LoginActivity.java

```diff
@@ -0,0 +1,50 @@
+package com.example.duccm.learningapp;
+
+import android.content.Intent;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.TextView;
+import android.widget.Toast;
+import com.example.duccm.learningapp.util.Utils;
+
+public class LoginActivity extends AppCompatActivity implements View.OnClickListener {
+    private Button buttonLogin;
+    private EditText editEmail;
+    private EditText editPassword;
+    private TextView textRegister;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_login);
+        buttonLogin = (Button) findViewById(R.id.button_log_in);
+        buttonLogin.setOnClickListener(this);
+        textRegister = (TextView) findViewById(R.id.text_register);
+        textRegister.setOnClickListener(this);
+        editEmail = (EditText) findViewById(R.id.edit_email);
+        editPassword = (EditText) findViewById(R.id.edit_password);
+    }
+
+    @Override
+    public void onClick(View v) {
+        switch(v.getId()) {
+            case R.id.button_log_in:
+                if (Utils.isValidEmail(editEmail.getText().toString()) &&
+                        editPassword.getText().toString().length() > 6) {
+                    Toast.makeText(LoginActivity.this, "Login Success", Toast.LENGTH_SHORT).show();
+                    Intent intent = new Intent(LoginActivity.this, MainActivity.class);
+                    startActivity(intent);
+                } else {
+                    Toast.makeText(LoginActivity.this, "Wrong email or password", Toast.LENGTH_SHORT).show();
+                }
+                break;
+            case R.id.text_register:
+                Intent intent = new Intent(LoginActivity.this, RegisterActivity.class);
+                startActivity(intent);
```

Comment:
có thể viết gọn luôn startActivity(new Intent(Login....this,Register.....class));